mathics/core/expression.py

```diff
@@ -538,6 +538,10 @@ def evaluate(
         return expr
 
     def evaluate_elements(self, evaluation) -> "Expression":
+        """
+        return a new expression with the same head, and the
+        evaluable elements evaluated.
```

Comment:
"evaluable" is jarring. Maybe we can just remove this word. 

From an old IBM manual: 

> Depressing the Power On key initiates the power on sequence. 
> Depressing the Power Off key initiates the power off sequence.
> Depressing the Power On key when the power is already on has no effect.
> Depressing the Power Off key when the power is already off has no effect.

Similarly trying to evaluation an element where there is nothing to evaluate has no effect, so suspect does not need to be spelled out.